// Copyright 2021 The MediaPipe Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto2";

package mediapipe;

import "mediapipe/framework/calculator.proto";

message RefineLandmarksFromHeatmapCalculatorOptions {
  extend mediapipe.CalculatorOptions {
    optional RefineLandmarksFromHeatmapCalculatorOptions ext = 362281653;
  }
  optional int32 kernel_size = 1 [default = 9];
  optional float min_confidence_to_refine = 2 [default = 0.5];
  optional bool refine_presence = 3 [default = false];
  optional bool refine_visibility = 4 [default = false];
}
